client/src/components/Sort/SortOptionsList.jsx

```diff
@@ -1,14 +1,23 @@
+// @flow
+
 import inlineStyles from "../../static/inline.css";
 import marginLeftStyles from "../../static/margin-left.css";
 
 import React from "react";
-import PropTypes from "prop-types";
 import SortOption from "./SortOption";
 import SORT_TYPE from "./sortType";
 
-const SortOptionsList = ({ currentSortType, onSwitchSortType }) => {
-  const sortOptions = Object.values(SORT_TYPE).map(sortType => (
-    <SortOption
+type SortOptionsListProps = {
+  currentSortType: string,
+  onSwitchSortType: function
+};
+
+const SortOptionsList = (sortOptionListProps:SortOptionsListProps) => {
+  const { currentSortType, onSwitchSortType } = sortOptionListProps;
+
+  const sortOptions = Object.values(SORT_TYPE).map((sortType:mixed) => (
+    (typeof sortType === 'string') &&
```

Comment:
_minor_

It's a redundant check since you're doing values mapping over a constant value.